### PR TITLE
need a review 

### PR DESCRIPTION
**Pull request by Estera94**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    width: 50%;
+    cursor: pointer;
+    font-size: 2.5rem;
+    color: white;
+    background-color: #e72424;
+    padding: 12px;
+}
+
+.submit:hover {
+    box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24), 0 17px 50px 0 rgba(0,0,0,0.19);
```
Comment (Alina1319) [inline css/style.css]: use hex colour

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+label {
+    color: #976d39;
+    margin: 20px;
+    font-size: 2.5rem;
+}
+
+.text {
+    color: red;
```
Comment (Alina1319) [inline css/style.css]: change the red colour

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    color: #585856;
+    font-weight: 600;
+}
+
+.subSection {
+    border: #b9b9b1 1px solid;
+    box-shadow: 13px 9px 15px -1px #3f3e3e52;
+    border-radius: 8px;
+    padding: 15px;
+    background-color:#FFFFFFFF;
```
Comment (nealioa) [inline css/style.css]: Add a space after the colon.

File: functions.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    foreach ($getCars as $cars) {
+         $display .= '<div class="subSection">'
+                        .'<p>'. 'Make: ' . '<span>'. $cars['make'] .'</span>' . '</p>'
+                        .'<p>'. 'Model: ' . '<span>'. $cars['model'] .'</span>' . '</p>'
+                        .'<p>'. 'Fuel Type: ' . '<span>'. $cars['fuel'] .'</span>' .'</p>'
+                        .'<p>'. 'Gearbox: ' . '<span>'. $cars['gearbox'] .'</span>' . '</p>'
+                        .'<p>'. 'Year: ' . '<span>'. $cars['year'] .'</span>' . '</p>'
+                   . '</div>';
+    }
+         $display .= '</div>';
```
Comment (nealioa) [inline functions.php:33]: Indented too far.

File: functions.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+         return $display;
```
Comment (nealioa) [inline functions.php:35]: Indented too far also.

File: functions.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+{
+    $db = new PDO('mysql:host=db;dbname=collector', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function getCarsDb(): array
+{
+    $db = connectToDB();
+    $query = $db->prepare('SELECT `make`,`model`,`fuel`,`gearbox`,`year` FROM `cars` JOIN `fuel` on `cars` . `fuel_type` = `fuel` . `id`');
```
Comment (nealioa) [inline functions.php]: This line is a bit long.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: carsCollection.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    <link rel="stylesheet" href=" css/style.css">
+    <!-- FONT -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;0,800;1,400&display=swap" rel="stylesheet">
+    <!-- PAGE TITLE -->
+    <title>Your Collection</title>
+</head>
+<body>
+<section>
+    <div class="button">
```
Comment (luciavelasco) [inline carsCollection.php:21]: The button (lines 21-23) is part of story 2; please remove the button from story1 branch (add it in story2 branch)

**Pull request merged**